app/assets/stylesheets/config/_fonts.scss

```diff
@@ -1,10 +1,44 @@
 // Import Google fonts
 @import url('https://fonts.googleapis.com/css?family=Nunito:400,700|Work+Sans:400,700&display=swap');
+@import url("https://use.typekit.net/hfn2egb.css");
 
 // Define fonts for body and headers
-$body-font: "Work Sans", "Helvetica", "sans-serif";
-$headers-font: "Nunito", "Helvetica", "sans-serif";
+$body-font: "area-normal", sans-serif, 20px;
 
+$headers-font: "area-normal", sans-serif, 40px;
+
+
+h1 {
+  font-family: "area-normal", sans-serif;
+font-weight: 700;
+font-style: normal;
+text-align: center;
+font-size: 40px;
+
```

Comment:
there is just some styling issues - please adjust indentation and delete the empty line